docs/adr/003-external-maker-decision-logic.md

```diff
@@ -0,0 +1,38 @@
+# ADR 003 - Keep maker decision logic external to the system
+
+```
+Author: @thomaseizinger
+Date: 29/12/2021
+Status: Active
+```
+
+## Summary
+
+Keep the maker's business decision logic for sending out offers or accepting / declining incoming orders outside the system.
+
+## Context
+
+There are multiple aspects contributing to this decision.
+
+### 1. Business logic is financially valuable
+
+All of itchysats is open-source to allow for auditability and independent verification.
```

Comment:
I think it could be worth adding a sentence that distinguishes what is mentioned as "business logic" from what we call "protocol". 

The business logic only allows decision making above the protocol level, i.e. the protocol is always ensured no matter what the external business logic is. 
For it this is clear, but for someone else reading through these docs it help.

Reply:
Feel free to send a follow-up patch that makes this clearer!

Reply:
We never explicitly talked about these so I just winged these :)